source/extensions/retry/host/omit_hosts/omit_hosts.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "envoy/upstream/retry.h"
+#include "envoy/upstream/upstream.h"
+
+#include "extensions/retry/host/well_known_names.h"
+
+namespace Envoy {
```

Comment:
do you mind adding some more namespaces here? I know the other extensions do this as well, but we usually have namespaces somewhat matching the directory structure (e.g. https://github.com/envoyproxy/envoy/blob/master/source/extensions/retry/priority/previous_priorities/config.h)

Reply:
Done